dotnet/src/Connectors/Connectors.OpenAI/TextToImage/AzureOpenAITextToImageService.cs

```diff
@@ -131,9 +131,8 @@ private static OpenAIClientOptions GetClientOptions(HttpClient? httpClient, stri
     {
         OpenAIClientOptions.ServiceVersion version = apiVersion switch
         {
-            // Dalle-E-3 is only supported in 2023-12-01-preview
-            "2023-12-01-preview" => OpenAIClientOptions.ServiceVersion.V2023_12_01_Preview,
-            _ => OpenAIClientOptions.ServiceVersion.V2023_12_01_Preview
+            // Dalle-E-3 is only supported post 2023-12-01-preview
+            _ => OpenAIClientOptions.ServiceVersion.V2024_02_15_Preview
```

Comment:
Why are we enforcing all to use the latest?

Suggest keeping possible to specify the previous version when it is explicitly requested.

Reply:
> OpenAIClientOptions.ServiceVersion.V2023_12_01_Preview

This no longer exists.

Reply:
Ok for context:

https://learn.microsoft.com/en-us/azure/ai-services/openai/api-version-deprecation

Retiring in 1 month, Approved, thanks.